src/database/queries/proposals/selectWithPagination.sql

```diff
@@ -4,5 +4,15 @@ SELECT p.id AS "id",
   p.opportunity_id AS "opportunityId",
   p.created_at AS "createdAt"
 FROM proposals p
+  LEFT JOIN proposal_versions pv ON pv.proposal_id = p.id
+  LEFT JOIN proposal_field_values pfv on pfv.proposal_version_id = pv.id
+WHERE
+  CASE
+    WHEN :search::text != '' THEN
+      pfv.value_search @@ websearch_to_tsquery(:search::text)
+    ELSE
+      true
+    END
+GROUP BY p.id
 ORDER BY p.id DESC
```

Comment:
Right now we're considering search results to be a filter, but probably we want to [rank the results](https://www.postgresql.org/docs/current/textsearch-controls.html#TEXTSEARCH-RANKING). This is another argument for splitting the search/nonsearch cases into separate queries. This is probably also future work, as I think we'd want to include the rank in the API response, and we don't need to blow up this PR or block on this suggested enhancement.

Reply:
I'll make an issue for this.

Reply:
Created issue #332